### PR TITLE
fix/#117 add time and date in messages

### LINKED ISSUE #117: Add time and date in messages
### Explication

In the original design, at chat view, each one of the messages has and image (dasher or user), text, and message's time. From database we receive `updated_at`, and we could use it to show date and time of messages.

### Screenshots
image

### Checklist

Describe the key points to complete the task. This points will help in the pull request test.

This issue can be closed when the following tasks are complete:

- [ ] Add message date and time below
- [ ] Show the time and date in a format understandable for humans, for example using:
 ```javascript
new Date(updated_at).toLocalString("es-CO")
```
- [ ] Add the correct styles for time and date

#### Figma design
https://www.figma.com/file/tbhMgDT10bbVjB9GazF97y/Dasher?node-id=771%3A593
#### Date object documentation
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString

### PR DESCRIPTION
**Pull request by arodriguezl9302**
### Description

Add time and date in messages

fixes #117 

#### Screenshots

image

#### Type of change

Bug fix (non-breaking change which fixes an issue)

### How to test it

1. Go to 'http://localhost:3000/chat/1'

### Checklist:

- [x] Add message date and time below
- [x] Show the time and date in a format understandable for humans, for example using: new Date(updated_at).toLocalString("es-CO")
- [x] Add the correct styles for time and date

### CODE REVIEW

**Comment by heliomar-pena** [comment]
In the screenshot I'm seeing the date, but there isn't the time, for example: 8:00pm, 3:00am. Can you check please?

**Review by e-azocar — COMMENTED** [review]
It's really good, but would be better with the message time

File: src/styles/colors.js [hunk 1]
```diff
@@ -32,6 +33,7 @@ const colors = {
   red01: '#F74326',
   purple10: '#3E254C',
   purple07: '#6E57C4',
+
```
Comment (Israel-Laguan) [inline src/styles/colors.js]: Nit: extra line

Comment (arodriguezl9302) [reply src/styles/colors.js]: ready

**Comment by arodriguezl9302** [comment]
### Done, this is how it looks after the changes
image

**Review by Israel-Laguan — APPROVED** [review]

**Review by e-azocar — APPROVED** [review]
It's ok, I liked how was before, but adding the message time. Something like this:
image

**Review by heliomar-pena — APPROVED** [review]
Good work!

**Comment by arodriguezl9302** [comment]
> 

I asked @katsu08 and he told me it was another issue

**Pull request merged**